pages/home_page.py

```diff
@@ -1,17 +1,11 @@
 from methods import Methods
 from locators import Locators
+from pages.header import Header
 
-class HomePage:
+class HomePage(Header):
```

Comment:
do you use there something from Header class?

Reply:
yep. So far this HomePage is not completed. More methods will be added later